docs/source/deep-learning/pytorch/quickstart/pytorch_quickstart.ipynb

```diff
@@ -0,0 +1,1361 @@
+    },
+    "2fd637fbe52e4d7f82c8f84512191346": {
+     "model_module": "@jupyter-widgets/base",
+     "model_name": "LayoutModel",
+     "model_module_version": "1.2.0",
+     "state": {
+      "_model_module": "@jupyter-widgets/base",
+      "_model_module_version": "1.2.0",
+      "_model_name": "LayoutModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/base",
+      "_view_module_version": "1.2.0",
+      "_view_name": "LayoutView",
+      "align_content": null,
+      "align_items": null,
+      "align_self": null,
+      "border": null,
+      "bottom": null,
+      "display": null,
+      "flex": null,
+      "flex_flow": null,
+      "grid_area": null,
+      "grid_auto_columns": null,
+      "grid_auto_flow": null,
+      "grid_auto_rows": null,
+      "grid_column": null,
+      "grid_gap": null,
+      "grid_row": null,
+      "grid_template_areas": null,
+      "grid_template_columns": null,
+      "grid_template_rows": null,
+      "height": null,
+      "justify_content": null,
+      "justify_items": null,
+      "left": null,
+      "margin": null,
+      "max_height": null,
+      "max_width": null,
+      "min_height": null,
+      "min_width": null,
+      "object_fit": null,
+      "object_position": null,
+      "order": null,
+      "overflow": null,
+      "overflow_x": null,
+      "overflow_y": null,
+      "padding": null,
+      "right": null,
+      "top": null,
+      "visibility": null,
+      "width": null
+     }
+    },
+    "ab28fa0960d64b5d83f2026d05e9c71d": {
+     "model_module": "@jupyter-widgets/controls",
+     "model_name": "DescriptionStyleModel",
+     "model_module_version": "1.5.0",
+     "state": {
+      "_model_module": "@jupyter-widgets/controls",
+      "_model_module_version": "1.5.0",
+      "_model_name": "DescriptionStyleModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/base",
+      "_view_module_version": "1.2.0",
+      "_view_name": "StyleView",
+      "description_width": ""
+     }
+    }
+   }
+  }
+ },
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Quickstart with MLflow PyTorch Flavor\n",
+    "\n",
+    "In this quickstart guide, we will walk you through how to log your PyTorch experiments to MLflow. After reading this quickstart, you will learn the basics of logging PyTorch experiments to MLflow, and how to view the experiment results in MLflow UI.\n",
+    "\n",
+    "This quickstart guide is compatible with cloud-based notebook such as Google Colab and Databricks notebook, you can also run it locally."
+   ],
+   "metadata": {
+    "id": "S9KwXx6Hz3yl"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Install Required Packages"
+   ],
+   "metadata": {
+    "id": "CZwvVcIg0Z1P"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "Sds9DYr9zcs0"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q mlflow torchmetrics torchinfo"
```

Comment:
We probably don't want to use the older shell-imperative command here; definitely not in Databricks. %pip is preferred (also for its better and nicer-looking stdout reporting)

Reply:
(depending on how many virtual environments you have, !pip can install to a different environment, which is super frustrating)

Reply:
that's a good point, I have been using `!` for years, and it's the right time to switch over I guess.